axum/CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **breaking:** Only inherit fallbacks for routers nested with `Router::nest`.
   Routers nested with `Router::nest_service` will no longer inherit fallbacks ([#1956])
 - **fixed:** Don't remove the `Sec-WebSocket-Key` header in `WebSocketUpgrade` ([#1972])
+- **added:** Extract `axum::extract::Query` from `&http::Uri` ([#2058])
```

Comment:
I think this should just say

```suggestion
- **added:** Add `axum::extract::Query::try_from_uri` ([#2058])
```

Since "extract" refers to extractors and `Query` was already an extractor.